test/parallel/test-child-process-stdin.js

```diff
@@ -34,8 +34,8 @@ cat.on('exit', common.mustCall(function(status) {
 
```

Comment:
There are two uses of `assert.ok()` in this file that could be greatly improved by changing to `assert.strictEqual()`.